tests/integration/test/plan/mocking_and_stubbing/mocking_test.pysoa

```diff
@@ -0,0 +1,95 @@
+test name: simple_mock_works
```

Comment:
This file was actually just moved from `tests/unit/test/plan/mocking_and_stubbing/mocking_test.pysoa`, but Git saw it as a rewrite.